scanpy/plotting/_docs.py

```diff
@@ -123,7 +123,20 @@
     as defined by matplotlib `scatter` function. When making multiple plots, vmin can
     be a list of values, one for each plot. For example `vmin=[0.1, 'p1', None, my_vmin]`
 vmax
-    Maximum value to plot. The format is the same as for `vmin`\
+    Maximum value to plot. The format is the same as for `vmin`.
```

Comment:
I don't think "Maximum value to plot" is an accurate description of what this does. This doesn't filter which points are plotted, it determines which value gets the color-maps highest value. There's probably a snippier way to phrase that.

"The value representing the center of the color scale" is better to me.

Reply:
How matplotlib does it:

```
vmin, vmax : float, default: None
    *vmin* and *vmax* are used in conjunction with the default norm to
    map the color array *c* to the colormap *cmap*. If None, the
    respective min and max of the color array is used.
    It is deprecated to use *vmin*/*vmax* when *norm* is given.
```